pdtf-api-1.2.0.yaml

```diff
@@ -0,0 +1,158 @@
+openapi: 3.0.3
+info:
+  title: Property Data Trust Framework API
+  version: 1.2.0
+  license:
+    name: MIT
+    url: https://opensource.org/licenses/MIT
+  description: This API defines how participants in the UK Property Data Trust Framework can securely share property data, together with the verifiable provenance of that data. The API is built around a variation of OpenID Connect Verified Claims schema.
+servers:
+  - url: http://api.pdtf-example.com
+    description: Example PDTF data source
+tags:
+  - name: transactions
+    description: Transactions API
+paths:
+  /transactions:
+    post:
+      tags:
+        - transactions
+      summary: Add claims on a transaction
+      operationId: addTransactionClaims
+      responses:
+        "201":
+          description: Created
+        "400":
+          description: Bad Request - the supplied claims contain one or more validation errors
+        "401":
+          description: Unauthorized
+      description: Add claims on a transaction
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: https://raw.githubusercontent.com/Property-Data-Trust-Framework/schemas/master/src/schemas/pdtf-transaction.json
+    get:
+      parameters:
+        - $ref: "#/components/parameters/uprn"
+      summary: Get a list of claims on a transaction
+      tags:
+        - transactions
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: https://raw.githubusercontent.com/Property-Data-Trust-Framework/schemas/master/src/schemas/pdtf-transaction.json
+      operationId: getTransactions
+  /transactions/{id}:
+    get:
+      parameters:
+        - $ref: "#/components/parameters/transactionIdInQueryParameter"
+      summary: Get a list of claims for a given transaction ID
+      tags:
+        - transactions
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: https://raw.githubusercontent.com/Property-Data-Trust-Framework/schemas/master/src/schemas/pdtf-transaction.json
+        "404":
+          description: No claims are available for this property
+      description: Get a list of claims for a given transaction ID
+      operationId: getTransactionsById
+  /transactions/propertyPack:
```

Comment:
`/propertyPack` is now one level down in the schema, alongside `participants` etc. Should this be `/transactions/{id}/state` or similar? Makes sense / is REST-ful to include the parameter in the path rather than as a query.

Reply:
The reasoning behind not putting the `{id}` in the path here is so that we could support multiple query parameters (and it returns one or more transactions)